### PR TITLE
Create journal-of-music-technology-and-education.csl

### PR DESCRIPTION
**Pull request by algomusic**
A new style for the Intellect journal of the same name.

### CODE REVIEW

File: journal-of-music-technology-and-education.csl [hunk 1]
```diff
@@ -0,0 +1,585 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Journal of Music Technology and Education</title>
+    <title-short>JMTE</title-short>
+    <id>http://www.zotero.org/styles/journal-of-music-technology-and-education</id>
+    <link href="http://www.zotero.org/styles/journal-of-music-technology-and-education" rel="self"/>
+    <link href="http://owl.english.purdue.edu/owl/resource/560/01/" rel="documentation"/>
```
Comment (adam3smith) [inline journal-of-music-technology-and-education.csl]: could you replace the link here with a link to the instructions for authors for JMTE, please?

File: journal-of-music-technology-and-education.csl [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+    <author>
+      <name>Andrew R. Brown</name>
+      <email>algorithmicmusic@gmail.com</email>
+    </author>
```
Comment (adam3smith) [inline journal-of-music-technology-and-education.csl]: add ISSN and (as applicable) EISSN here please

File: journal-of-music-technology-and-education.csl [hunk 3]
```diff
@@ -0,0 +1,585 @@
...
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
```
Comment (adam3smith) [inline journal-of-music-technology-and-education.csl]: we'd like this replaced by one of the non-generic categories here http://docs.citationstyles.org/en/latest/specification.html#appendix-i-categories maybe humanities and/or engineering?

**Comment by adam3smith** [comment]
Looks good otherwise. You can update your pull request using the pencil icon here: https://github.com/citation-style-language/styles/pull/1731/files
let us know when you're done, we don't get notifications on updated pull requests.

**Comment by rmzelle** [comment]
@algomusic, let us know if you need help with this.

**Comment by rmzelle** [comment]
Merged by hand with remaining changes.

**Pull request closed without merging**